### ISSUE TITLE
npm package of latest version corrupted?

### ISSUE THREAD

**Issue by blanser-cc** [wiremock/wiremock-npm#2]
When I try to install the latest version, I get the following error that to me looks like there is something wrong with the package:
```
% npm install wiremock       
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path .../node_modules/wiremock/jdeploy-bundle/jdeploy.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '.../node_modules/wiremock/jdeploy-bundle/jdeploy.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     .../.npm/_logs/2021-05-16T20_34_56_690Z-debug.log
```
Version 2.27.2 I can install and run just fine.

**Comment by mathew-ferry13** [comment]
Receiving the same error

**Comment by tomasbjerre** [comment]
Seems like the jar is not included in the 2.80.0 npm package... I'm working on it.

**Comment by tomasbjerre** [comment]
I'm not really sure how to handle this situation.

The release is automated with Travis. And the release script did not fail when it failed to download the wiremock jar. It published a npm package without any jar in it. This should not happen again, i patched the [release script](https://github.com/tomasbjerre/wiremock-npm/blob/master/release.sh#L6).

But now there is a version `2.28.0` that does not work. And if I publish `2.28.1` to fix it, the versioning will be out of sync with Wiremock... not an option.

Perhaps deprecating `2.28.0`. But that may indicate that this entire package is deprecated?

Removing `2.28.0` would be my preferred option, but that does not seem possible 24h after original release.

**Comment by tomasbjerre** [comment]
I released wiremock 2.27.2 as this package in version 2.28.1 now. Temporary fix.
